packages/shared/src/graphql/notifications.ts

```diff
@@ -41,6 +47,33 @@ export interface NotificationsData {
   notifications: Connection<Notification>;
 }
 
+export enum NotificationIcon {
```

Comment:
I dont think these should be inside the graphql directory, can you move them to a utilities directory?

Reply:
This is part of the returned value of the GraphQL query. Do you think we should take this out?

Reply:
I agree with Eliz. We use it throughout. Let's take it to somewhere else

Reply:
Alright. In that case, let me turn the icon property into just a string as it might result to a dependency cycle if the utils file will ever need anything inside from this file.

Reply:
I think enum is ok for this purpose. Better than a string actually. I wouldn't worry about cycle at this point